### PR TITLE
Cookie refresh deployment

### PR DESCRIPTION
**Pull request by markjamesbutler**
This PR moves the cookie refresh logic from the existing a/b test of 75% of signed-in users to all signed-in users.  

Any users that are signed-in and have cookies older than one month will have their cookies 'refreshed'.  This means the cookies are re-issued thus implementing persistent sign-in for active users. 

[Trello ticket](https://trello.com/c/kszkBW8N/225-2-persistent-sign-in-for-existing-users)

### CODE REVIEW

File: common/app/conf/switches.scala [hunk 1]
```diff
@@ -789,6 +788,15 @@ object Switches {
     exposeClientSide = false
   )
 
+  val IdentityCookieRefreshSwitch = Switch(
```
Comment (johnduffell) [inline common/app/conf/switches.scala]: worth considering deleting the switch altogether if it's not something we'll want to tweak in future

**Comment by johnduffell** [comment]
this is great! :+1:

File: common/app/conf/switches.scala (hunk 1 above)
Comment (markjamesbutler) [inline common/app/conf/switches.scala]: Yes, I'd rather keep this around for now (just in-case we see unforeseen issues).

Comment (johnduffell) [inline common/app/conf/switches.scala]: fair enough, sounds sensible - in that case worth putting an expiry on it

Comment (johnduffell) [inline common/app/conf/switches.scala]: actually you already merged so probably not worth it this time!

**Pull request merged**